vernac/comProgramFixpoint.ml

```diff
@@ -237,15 +233,13 @@ let do_program_recursive ~pm ~scope ?clearbody ~poly ?typing_flags ?user_warns ?
   let (fixnames,fixrs,fixdefs,fixtypes) = fix in
   let collect_evars name def typ impargs =
     (* Generalize by the recursive prototypes  *)
-    let terms = [def; typ] in
-    let using = Option.map (fun using -> Proof_using.definition_using env evd ~fixnames ~using ~terms) using in
```

Comment:
Looking better, I see that `using` is here lost on `Program Fixpoint`. I propose to pass it instead to `add_mutual_definitions` which should itself be modified so as to pass it to `prg_using`. 

Incidentally, this is related to a problem I had also in #18742, namely that `using` should be global to the block of mutual fixpoints.

Reply:
Actually I think that "delegation" is disabled for mutual fixpoints, (and not sure about Program, not sure if that would work).

I'll see what I can do.

Reply:
Should be fixed.